Integrations/SplunkPy/SplunkPy.py

```diff
@@ -187,7 +187,7 @@ def notable_to_incident(event):
     if demisto.get(event, "_time"):
         incident["occurred"] = event["_time"]
     else:
-        incident["occurred"] = datetime.now()
+        incident["occurred"] = datetime.now().strftime('%Y-%m-%dT%H:%M:%S.0+00:00')
```

Comment:
What is the `0+00:00` after the `%S`?

Reply:
Seems to be needed after checking and testing the bug. so thay Splunk will be able to parse the timestamp